website/content/docs/k8s/connect/terminating-gateways.mdx

```diff
@@ -140,49 +136,82 @@ service "example-https" {
 
 ```shell-session
 $ consul acl policy create -name "example-https-write-policy" -rules @write-policy.hcl
+ID:           xxxxxxxxxxxxxxx
+Name:         example-https-write-policy
+Description:
+Datacenters:
+Rules:
+service "example-https" {
+  policy = "write"
+}
 ```
 
 Now fetch the id of the terminating gateway token
 
 ```shell-session
-$ consul acl token list | grep terminating-gateway-terminating-gateway-token
```

Comment:
Previously this would print the ID of the policy which couldn't be used in the next command. Instead we need the ID of the token.